File: backend/src/main/java/com/xxgwy/common/security/JwtConfig.java
package com.xxgwy.common.security;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@Data
@Configuration
@ConfigurationProperties(prefix = "app.jwt")
public class JwtConfig {
    private String secret = "change-me-in-production-change-me-in-production!!";
    private long expirationHours = 24;
    private long refreshExpirationHours = 168; // 7 days
}

File: backend/src/main/kotlin/com/xxgwy/common/security/JwtConfig.kt
package com.xxgwy.common.security

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties(prefix = "app.jwt")
class JwtConfig {
    var secret: String = "change-me-in-production-change-me-in-production!!"
    var expirationHours: Long = 24
    var refreshExpirationHours: Long = 168
}
